Refresh English source texts

gerhardol opened the pull request:
By @Miniangel merge of suggested parts in #926 #927 #928 

About 50 strings changed, will need to be updated in all translations...

gerhardol commented on common/src/main/res/values/strings.xml:
```diff
@@ -266,8 +262,8 @@
   <string name="Add_workout_notes">Add workout notes</string>
-  <string name="Pruning_deleted_activities_from_database">Prune deleted activities from the database</string>
-  <string name="Prune">Prune</string>
+  <string name="Pruning_deleted_activities_from_database">Permanently delete deleted activities</string>
```
delete deleted
Better?

Miniangel replied:
Yeah, this string is giving me a headache. I mean, the original _prune_ was not completely bad, and the **delete deleted** sounds a bit weird. What do you think about something like **Clean deleted activities**?

gerhardol replied:
ok

Miniangel commented:
Hi,

I have taken a look at the changes and everything looks good to me. About
the Catalan translation you do not need to worry, I have taken care of it.
Concerning other languages, yes, they should be verified, but I do not
think it is an urgent matter because the changes we have applied are
details, I mean, the global meaning of sentences, in general, remains the
same.

Do I have to do anything else? I am new to all this stuff, so if there is
something left that I must do, please let me know.

Greetings,

Missatge de Gerhard Olsson <notifications@github.com> del dia dt., 30 de
juny 2020 a les 22:44:

> By @Miniangel <https://github.com/Miniangel> merge of suggested parts in
> #926 <https://github.com/jonasoreland/runnerup/pull/926> #927
> <https://github.com/jonasoreland/runnerup/pull/927> #928
> <https://github.com/jonasoreland/runnerup/pull/928>
>
> About 50 strings changed, will need to be updated in all translations...
> ------------------------------
> You can view, comment on, or merge this pull request online at:
>
>   https://github.com/jonasoreland/runnerup/pull/930
> Commit Summary
>
>    - Update strings.xml
>    - Adjust suggested translations
>
> File Changes
>
>    - *M* app/res/values/cues.xml
>    <https://github.com/jonasoreland/runnerup/pull/930/files#diff-962e574cfe64f22731ac3705f3bf84e0>
>    (4)
>    - *M* app/src/main/org/runnerup/db/DBHelper.java
>    <https://github.com/jonasoreland/runnerup/pull/930/files#diff-8915724e8f4c826a945603a66151f5ef>
>    (8)
>    - *M* app/src/main/org/runnerup/view/HRSettingsActivity.java
>    <https://github.com/jonasoreland/runnerup/pull/930/files#diff-55e6029bbf8842e1614533f019edfc7a>
>    (2)
>    - *M* app/src/main/org/runnerup/view/ManageWorkoutsActivity.java
>    <https://github.com/jonasoreland/runnerup/pull/930/files#diff-984e5b72e44c8ba377a8744cf53995f8>
>    (4)
>    - *M* app/src/main/org/runnerup/view/SettingsActivity.java
>    <https://github.com/jonasoreland/runnerup/pull/930/files#diff-303d88c303220688eedd97ffb2a68de3>
>    (4)
>    - *M* common/src/main/res/values/array.xml
>    <https://github.com/jonasoreland/runnerup/pull/930/files#diff-87fa1cf5a04e8271f5054d635f733267>
>    (2)
>    - *M* common/src/main/res/values/strings.xml
>    <https://github.com/jonasoreland/runnerup/pull/930/files#diff-94517e5911398231e0e28d398ad19fa9>
>    (86)
>
> Patch Links:
>
>    - https://github.com/jonasoreland/runnerup/pull/930.patch
>    - https://github.com/jonasoreland/runnerup/pull/930.diff
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jonasoreland/runnerup/pull/930>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQDVOI2YRSI6F2WHYGSFMR3RZJFC7ANCNFSM4OMULVOA>
> .
>


-- 
Miquel-Àngel B. i F.

gerhardol commented:
Squashed and removed single sentences ending with period, to be consistent.

> I have taken a look at the changes and everything looks good to me. About
the Catalan translation you do not need to worry, I have taken care of it.
Concerning other languages, yes, they should be verified, but I do not
think it is an urgent matter because the changes we have applied are
details, I mean, the global meaning of sentences, in general, remains the
same.

The problem is that every language has to be actively updated and many may remain untranslated.
Unless I manually retain the current translation every time pulling from Transifex

> Do I have to do anything else? I am new to all this stuff, so if there is
something left that I must do, please let me know.

Since created a new PR, I handle this. PRs should be reviewed and you are welcome to do that, but that would just be a formal change, approved in text.

If you know any other languages, it would be good if you could translate them. (I would be surprised if you are not good in Spanish too for instance.)

But the update process will take some hours, cannot do this tonight so will not merge yet.

Miniangel commented:
OK, then I am going to take a look at French, Italian, Romanian, and
Spanish (if they exist). Those are the languages I can handle to some
degree. But someone should review what I do, as I do not work with these
languages at the same level as Catalan or English.

Missatge de Gerhard Olsson <notifications@github.com> del dia dc., 1 de
jul. 2020 a les 22:47:

> Squashed and removed single sentences ending with period, to be consistent.
>
> I have taken a look at the changes and everything looks good to me. About
> the Catalan translation you do not need to worry, I have taken care of it.
> Concerning other languages, yes, they should be verified, but I do not
> think it is an urgent matter because the changes we have applied are
> details, I mean, the global meaning of sentences, in general, remains the
> same.
>
> The problem is that every language has to be actively updated and many may
> remain untranslated.
> Unless I manually retain the current translation every time pulling from
> Transifex
>
> Do I have to do anything else? I am new to all this stuff, so if there is
> something left that I must do, please let me know.
>
> Since created a new PR, I handle this. PRs should be reviewed and you are
> welcome to do that, but that would just be a formal change, approved in
> text.
>
> If you know any other languages, it would be good if you could translate
> them. (I would be surprised if you are not good in Spanish too for
> instance.)
>
> But the update process will take some hours, cannot do this tonight so
> will not merge yet.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jonasoreland/runnerup/pull/930#issuecomment-652638572>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQDVOI7IXFLKB7VUP5GNLVTRZOOGJANCNFSM4OMULVOA>
> .
>


-- 
Miquel-Àngel B. i F.

gerhardol commented:
https://github.com/jonasoreland/runnerup/releases/tag/v2.0.6.3

@Miniangel Thanks, changes included in release

Miniangel commented:
Great, it looks very nice now! Anything else I can do?

Miquel-Àngel B. i F.

El ds., 4 jul. 2020, 23:18, Gerhard Olsson <notifications@github.com> va
escriure:

> https://github.com/jonasoreland/runnerup/releases/tag/v2.0.6.3
>
> @Miniangel <https://github.com/Miniangel> Thanks, changes included in
> release
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jonasoreland/runnerup/pull/930#issuecomment-653813869>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQDVOI3JXSXWN2J4GYL3MG3RZ6MBNANCNFSM4OMULVOA>
> .
>

gerhardol commented:
> Great, it looks very nice now! Anything else I can do?

I forgot to add Romanian audio cues, will update later?. Is it correct to write 'Română'?
Otherwise thanks!

Miniangel commented:
All right, tell me when it needs to be done! Română is indeed correct.
Thanks to you too, great work!

Miquel-Àngel B. i F.

El dg., 5 jul. 2020, 12:31, Gerhard Olsson <notifications@github.com> va
escriure:

> Great, it looks very nice now! Anything else I can do?
>
> I forgot to add Romanian audio cues, will update later?. Is it correct to
> write 'Română'?
> Otherwise thanks!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jonasoreland/runnerup/pull/930#issuecomment-653870937>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AQDVOI4XUTGM5UFCXT2LZ4TR2BI6RANCNFSM4OMULVOA>
> .
>
